ZenPacks/zenoss/Import4/events.py

```diff
@@ -53,6 +53,7 @@ def _check_files(self):
 
         if not os.path.isdir(self.zenbackup_dir):
             log.error('Backup directory does not exist. Run `check` command to extract the backup file.')
+            self.reportError('events_import', 'Events database error')
```

Comment:
I am not sure here.
That level of detail may best fit inside of a running log context. Whereas in a UI, they need to know there is an error so that they may be directed to the log file to figure out.